### DISCUSSION TITLE
🚀 Feature Request: Check rate limit state (e.g. how many requests/operations are left)

### DISCUSSION THREAD

**Discussion by brillout** [cloudflare/workers-sdk#13529] [Feature Requests]
### Describe the solution

Is there a way to check how many requests are left before hitting a rate limit?

In my case the rate limits I want to check are these:

> You have exceeded the daily Cloudflare Workers KV free tier limit of 1000 Workers KV delete operations.
> You have exceeded the daily Cloudflare Workers KV free tier limit of 1000 Workers KV put operations.

Context: I'm a maintainer of [Vike](https://vike.dev/) (a React/Vue/Solid frontend framework) and Vike's CI is testing Vike's Cloudflare Workers integration. When the number of operations left are close to be depleted, then the CI should skip the integration test.

I've looked at `$ wrangler -h` and https://developers.cloudflare.com/workers/platform/limits/ &mdash; I couldn't find a way to check the current state of the rate limit. So I'm guessing this is a feature request.

**Comment by towfiqi** [comment]
Any update? Is this now possible?
